.github/workflows/docker-publish-ecr.yml

```diff
@@ -34,5 +34,8 @@ jobs:
           docker build -t consoleme .
       - name: Tag, and push image to Amazon ECR
         run: |
+          export CONSOLEME_VERSION=$(python setup.py -q --version)
           docker tag consoleme:latest public.ecr.aws/consoleme/consoleme:latest
+          docker tag consoleme:latest public.ecr.aws/consoleme/consoleme:$CONSOLEME_VERSION
```

Comment:
I also like to add major and minor tags, for example releasing `v1.2.3` would create
* `consoleme:v1.2.3`
* `consoleme:v1.2`
* `consoleme:v1`

This lets folks stay up to date with new releases, but only within certain bounds.

Reply:
Great idea! I migrated this to a new PR - https://github.com/Netflix/consoleme/pull/9191